src/extension.ts

```diff
@@ -107,13 +102,17 @@ function changeConfig(params: vscode.ConfigurationChangeEvent) {
     if (need_to_restart_server) {
         if (g_languageClient !== null) {
             g_languageClient.stop()
-            setLanguageClient(null)
+            setLanguageClient()
         }
         startLanguageServer()
     }
 }
 
 async function startLanguageServer() {
+    const startupNotification = vscode.window.createStatusBarItem()
+    startupNotification.text = 'Starting Julia Language Server...'
+    startupNotification.show()

```

Comment:
I had this created at the original place so that it shows up as soon as the extension starts up. I think moving this down here might potentially mean that it will take a while until this element pops up in the UI?

Agreed that we don't need to have this as a global, but maybe it could be created in the old place, and then we do `startLanguageServer(startupNotification)`, i.e. just pass the element that was created in `activate` to the `startLanguageServer` function?

Reply:
>  think moving this down here might potentially mean that it will take a while until this element pops up in the UI?

um, I don't feel any delay because of that; after all what we do in `activate` before calling `startLanguageServer` is registering commands/telemetry stuff and activate child functions.

I also would like to point out this notification is essentially tied to LS start up, not to extension, and we want to see that for each `startLanguageServer` call, rather than the very first `activate` call ? It can happen when we change julia path.

Reply:
Sounds good!